### DISCUSSION TITLE
iLogtail 2.0版本采集配置不兼容变更说明

### DISCUSSION THREAD

**Discussion by henryzhx8** [alibaba/ilogtail#1294] [Announcements]
# 背景

随着iLogtail配置管理模块的升级改造完成，iLogtail也即将进入2.0时代。相比于1.x版本，2.0版本全面支持流水线化的日志处理方式，原有高性能加速插件经过改造，升级为多个原生处理插件，因此不再受限于只能在往SLS发送时使用，社区可以进一步享受原有加速插件所带来的高性能体验。

因此，原有的采集配置也需要进行更新。尽管社区已经尽量做到向前兼容，但是少量不兼容的改动仍然在所难免。如果您之前使用iLogtail采集文件，或者使用了加速插件，那么您需要在升级2.0版本前对采集配置进行微调，以符合2.0的采集配置要求。

# 方案

详见[采集配置不兼容升级方案](https://github.com/alibaba/ilogtail/files/13783931/default.pdf)

**Comment by LomotHo** [comment]
我看到2.0的配置变更文档里面要求输出插件列表必须包含flusher_sls输出插件，且flusher_sls中必须包含Logstore参数。这里的原因是什么呢
